src/core/pointcloud/qgslazinfo.h

```diff
@@ -102,6 +102,11 @@ class CORE_EXPORT QgsLazInfo
     //! Returns the number of extrabytes contained in the LAZ dataset
     int extrabytesCount() const { return mHeader.ebCount(); }
 
+    //! Returns the absolute offset to the first extended point record in the LAZ file
+    uint64_t firstExtendedPointRecordOffset() const { return mHeader.evlr_offset; }
```

Comment:
extended point record offset? I guess that should read "extended variable length record"?

By the way, here (and in the next function) I would just go with "evlr" acronym rather than full length names :)